README.md

```diff
@@ -1,113 +1,93 @@
 
-### GenAI models
+## Why use Genkit?
 
-**Unified API for generation** across AI models built by Google (Gemini, Gemma) and third party providers. Supports multimodal input, multimedia content generation, and custom options.
+Integrating generative AI into apps is complicated. You may need to tie together various components like generative AI models (LLMs), prompts, vector stores, embedders, and tools - each with provider-specific implementations and quirks. The non-deterministic nature of generative AI models also makes refining AI apps for production challenging.
 
-**Generate structured output** as strongly-typed objects with custom schemas for easy integration into your app.
+Genkit libraries provide lightweight abstractions and unified APIs that make it easy to define and compose AI app components from any provider into cohesive workflows with built-in type-safety and observability. Genkit's integrated tooling accelerates your iteration speed by helping you quickly test, visualize, and debug your app's AI logic.
 
-**Define custom tools for your AI models** to fetch data, display UI, write to a database, and more.
+## Get started
 
-**Write well structured prompts** with all relevant metadata encapsulated in a single [dotprompt](https://firebase.google.com/docs/genkit/dotprompt) file. Supports handlebars templating, history, multimedia, and more.
+- [Genkit TypeScript (Node.js) documentation](https://firebase.google.com/docs/genkit/get-started)
+- [Genkit Go documentation](https://github.com/firebase/genkit/blob/main/docs-go/get-started-go.md)
 
-![Screenshot of IDE showing Firebase Genkit RAG sample code](docs/resources/readme-rag-screenshot.png)
+> [!NOTE]
+> Genkit for Go is currently in Alpha, so we only recommend using it for prototyping and exploration. 
 
-### Retrieval
+## Framework key features
 
-**Build context-aware AI features** by indexing your data and dynamically retrieving relevant information from your database. Genkit provides flexible, light-weight abstractions for indexers and retrievers that work with any database provider.
+- **Unified APIs for generation and retrieval:** Use standardized APIs to generate, embed, index, and retrieve content with multiple AI model and vector store providers. The generation API supports streaming, structured output with custom schemas, multimodal input and output, function calling with tools, and custom configuration options. Make your code more flexible without sacrificing any provider-specific capabilities.
 
-### Evaluation
+- **Flows for multistep logic:** Write your AI app logic within observable, type-safe, streamable functions, called [flows](https://firebase.google.com/docs/genkit/flows). Flows are just like regular functions, but are integrated with Genkit tooling, group traces for multiple steps, are deployable as HTTP endpoints with minimal boilerplate, and have additional properties that make them well-suited to working with generative AI APIs. 
```

Comment:
Or, "Flows are just like regular functions, but have additional properties that make them well-suited to working with generative AI APIs. For instance, they are tightly integrated with Genkit's tooling and developer UI, with built-in observability for debugging and monitoring. When you're ready to deploy, Genkit flows are deployable as HTTP endpoints with minimal boilerplate."